flowzone.yml

```diff
@@ -1249,22 +1248,25 @@ jobs:
           driver-opts: network=host
           install: true
 
+      # set env var TARGET that is just a sanitized matrix.target
+      # set env var PREFIX if target is not 'default'
+      # set env var PLATFORM that is just a sanitized matrix.platform
+      # write compose yaml to file and set platform={{matrix.platform}} on any services with the 'build' directive
+      # write docker bake json to file
       - name: Set env vars
```

Comment:
@klutchell also breaks [here](https://github.com/balena-io/open-balena-vpn/actions/runs/4000612249/jobs/6866051809#step:6:80).

Reply:
I just noticed that too and this fix is [here](https://github.com/product-os/flowzone/pull/448)